config/settings.yml

```diff
@@ -1276,8 +1274,6 @@
         :poll: 1.seconds
       :event_catcher_openshift:
         :poll: 1.seconds
-      :event_catcher_openshift_enterprise:
-        :poll: 1.seconds
       :event_catcher_atomic_enterprise:
```

Comment:
@zeari ah, this is a leftover that is worth removing as well (cc @zgalor)

Check if there are other `atomic_enterprise` to drop.